src/ci/bin/test.inc.sh

```diff
@@ -1356,12 +1359,12 @@ cromwell::build::setup_common_environment() {
     esac
 
     cromwell::private::setup_secure_resources
+    cromwell::private::start_build_heartbeat
```

Comment:
This, and the related changes, feel like a refactor to me, but I'm not familiar enough with the code to know if there's a functional difference. Anything worth noting about this?

Reply:
Printing out heartbeats is another bandaid.

I'm still unsure why a couple of sbt tests mentioned in BT-140 just stopped responding for 10 minutes. In one analysis, there was another strange pause for 7 minutes before sbt just kept moving. GC pause? Virus scanner? Solar flare?

So, seeing that sbt was just pausing for 7 minutes at a time and continuing perhaps it just needed to take a nap and keep going. So the heartbeats that used to just be part of centaur tests are now part of \*all\* CI tests.